File: Tatarstan/Bars.GkhDi.Regions.Tatarstan/libs/B4/model/service/ProviderService.js
Ext.define('B4.model.service.ProviderService', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'ProviderService'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'BaseService', defaultValue: null },
        { name: 'Provider', defaultValue: null },
        { name: 'Name' },
        { name: 'DateStartContract', defaultValue: null },
        { name: 'Description' },
        { name: 'IsActive', defaultValue: true },
        { name: 'NumberContract' }
    ]
});

File: Bars.GkhDI/Entities/Service/ProviderService.cs
namespace Bars.GkhDi.Entities
{
    using System;
    using Gkh.Entities;

    /// <summary>
    /// Поставщики услуги
    /// </summary>
    public class ProviderService : BaseGkhEntity
    {
        /// <summary>
        /// Базовая услуга
        /// </summary>
        public virtual BaseService BaseService { get; set; }

        /// <summary>
        /// Поставщик услуги
        /// </summary>
        public virtual Contragent Provider { get; set; }

        /// <summary>
        /// Дата заключения договора
        /// </summary>
        public virtual DateTime? DateStartContract { get; set; }

        /// <summary>
        /// Примечание
        /// </summary>
        public virtual string Description { get; set; }

        /// <summary>
        /// Активный
        /// </summary>
        public virtual bool IsActive { get; set; }

        /// <summary>
        /// Номер договора
        /// </summary>
        public virtual string NumberContract { get; set; }
    }
}
